### ISSUE TITLE
Confusion regarding screen and window coordinates

### ISSUE THREAD

**Issue by wistpotion** [britzl/defold-orthographic#59]
Minor notice that screen coordinates use `action.x` instead of `action.screen_x`. This is a bit confusing, so unless there are other reasons to keep the current naming convention (other than backwards compatibility) it would be nice to switch the two names.

**Comment by britzl** [comment]
I agree. It is confusing. I'll try to fix it for the next release I make!

**Comment by britzl** [comment]
I've removed everything but `ortho.screen_to_world()` and `ortho.world_to_screen() ` with `ortho.screen_to_world()` expecting screen_x and screen_y, just like the standard camera API.
